components/pages/TodoList/index.js

```diff
@@ -0,0 +1,133 @@
+// TodoList.js
```

Comment:
This TodoList folder can be deleted, because this component has been moved to widgets folder